crates/shared/src/sources.rs

```diff
@@ -39,6 +41,7 @@ pub fn defaults_for_chain(chain_id: u64) -> Result<Vec<BaselineSource>> {
             BaselineSource::SushiSwap,
             BaselineSource::BalancerV2,
             BaselineSource::ZeroEx,
+            BaselineSource::UniswapV3,
```

Comment:
We can also add it as a default liquidity source for Görli.

Reply:
Ah yees, missed that one. Added.